files/migrations/0022_hash_existing_passwords.py

```
@@ -0,0 +1,38 @@
+        media.password = make_password(media.password)
+        media.save(update_fields=["password"])
+        updated += 1
+
+    if updated:
+        print(f"  Hashed {updated} media password(s)")
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("files", "0021_widen_password_field"),
+    ]
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)